README.rst

```diff
@@ -50,6 +57,34 @@ Example analysis script
 There are a number of tutorials_ on the MDAnalysis homepage that explain
 how to conduct RMSD calculations, Alignment and many more features of MDAnalysis.
 
+Example Trajectories-analysis script
```

Comment:
We intentionally kept the README small and we don't want to add more examples. Don't include it.